### PR TITLE
py-libsonata: add v0.1.29

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Review by wdconinc — COMMENTED** [review]
v0.1.29 requires python@3.9:, per https://github.com/BlueBrain/libsonata/commit/588434f8bfe5ff87331b1ea1b81697d55cc30f7d#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

**Comment by matz-e** [comment]
Thanks, fixed that!

File: var/spack/repos/builtin/packages/py-libsonata/package.py [hunk 1]
```diff
@@ -16,10 +16,13 @@ class PyLibsonata(PythonPackage):
     maintainers("tristan0x")
 
     version("master", branch="master")
+    version("0.1.29", sha256="321878f28c7d64a65683443f832dfa2f21ff8ed69e700d2dde62ccb5f87d4525")
     version("0.1.25", sha256="b332efa718123ee265263e1583a5998eaa945a13b8a22903873764cf1d8173fa")
 
     depends_on("cxx", type="build")  # generated
 
+    depends_on("python@3.9:", type=("build", "run"), when="0.1.29:")
```
Comment (wdconinc) [inline var/spack/repos/builtin/packages/py-libsonata/package.py]:
```suggestion
    depends_on("python@3.9:", type=("build", "run"), when="@0.1.29:")
```

**Review by wdconinc — APPROVED** [review]
Verified checksum. Verified upstream setup.py and pyproject.toml diff. Looks good to me.

**Comment by wdconinc** [comment]
@spackbot run pipeline

**Pull request merged**